core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -460,13 +463,13 @@ export class ContributionsAndReview
 
   loadContributions(shouldResetOffset: boolean):
     Promise<GetOpportunitiesResponse> {
-    if (!this.activeTabType || !this.activeSuggestionType) {
+    if (!this.activeTabType || !this.activeSubType) {
```

Comment:
"type" and "subtype" are in the same category, so if you have "activeTabType" you should have "activeTabSubtype" as well. Note that "subtype" is one word.